CONTRIBUTING.md

```diff
@@ -1,10 +1,37 @@
-Contributions to this repository are intended to become part of Recommendation-track documents governed by the
-[W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy/) and
-[Software and Document License](https://www.w3.org/Consortium/Legal/copyright-software).
-To make substantive contributions to specifications, you must either participate
-in the relevant W3C Working Group or make a non-member patent licensing commitment.
-If you are not the sole contributor to a contribution (pull request), please identify all 
-contributors in the pull request comment.
+# Contributing to the Web Neural Network API
+The Working Group welcomes contributions as an issue feedback or a pull request. For contributions aimed at making changes in the specification via a pull request, the following guidelines and best practices should be followed.
+
+## Goal
+Members of the Working Group contribute their work and time on a volunteering basis. Our goal is to provide a process that expedites proposing and reviewing changes to the specification within the scope of the [charter](https://www.w3.org/2023/04/web-machine-learning-charter.html) that results in clear and consistent documentation while reducing the opportunities for accidental mistakes and misinterpretation.
+
+## Type of change
```

Comment:
I personally like the category names introduced in this PR because they're more generic and as such familiar to a broader group of OSS contributors. We could consider mapping these to the W3C Process defined [classes of changes](https://www.w3.org/2021/Process-20211102/#correction-classes) that are more W3C specific and introduce both the W3C Process and Patent Policy linkage. Another possible W3C Process alignment is to speak about [substantive change](https://www.w3.org/2021/Process-20211102/#substantive-change) rather than material impact. Established W3C terminology is to speak of normative vs. informative or non-normative content (W3C Process does not have formal definitions for these terms so the generic meaning is assumed).

Using W3C parlance in this guideline doc might be too much so I'm not pushing that strongly. @dontcallmedom WDYT?